### DISCUSSION TITLE
excerpt/hkyhj/

### DISCUSSION THREAD

**Discussion by giscus** [Shitao5/shitao-blog#57] [Comments]
# excerpt/hkyhj/

想要把握这个时代，就必须理解计算机。理解计算机的关键，则是要理解计算机背后的人。表面上这是一个机器的时代，但是实际上机器的设计者决定了我们的时代。程序员的审美决定了你看到的软件界面，程序员的爱好决定了你有什么样的软件可以使用。我们的时代是程序员主导的时代，而伟大的程序员就是黑客。
[&hellip;] “黑客”这个词不仅是第一流能力的象征，还包含着求解问题过程中产生的精神愉悦或享受。也就是说，从一 &hellip;

https://shitao5.org/excerpt/hkyhj/

**Comment by LoganQiu** [comment]
吴兄（可否这么称呼？）
我发现你的这些书摘并没有在博客中展示出来是有什么原因么😌？顺便想请问书摘卡片是保存在 nas 挂载到 Alist 吗？

**Reply by Shitao5** [reply]
您好呀！当然可以哈哈。

书摘全部来自书本原句，没有个人思考的部分，因此最近开始的书摘便没有直接放在博客中展示，后续如有读后感则会引用这些未展示的内容，这居然还被你发现了哈哈！

书摘卡片直接存在云服务器的硬盘上，Alist 可以直接挂载本地磁盘，这样只需要一台云服务器就可以了。
